_posts/2021-06-03-opensearch-release-candidate-announcement.markdown

```diff
@@ -0,0 +1,68 @@
+- [SQL ODBC Driver](https://github.com/opensearch-project/sql/blob/main/release-notes/opensearch-sql.release-notes-1.0.0.0-rc1.md)
+- [Notebooks OpenSearch Plugin](https://github.com/opensearch-project/dashboards-notebooks/blob/main/release-notes/opensearch-dashboards-notebooks.release-notes-1.0.0.0-rc1.md)
+- [Notebooks Dashboards Plugin](https://github.com/opensearch-project/dashboards-notebooks/blob/main/release-notes/opensearch-dashboards-notebooks.release-notes-1.0.0.0-rc1.md)
+- [Reporting Opensearch Plugin](https://github.com/opensearch-project/dashboards-reports/blob/main/release-notes/opensearch-dashboards-reports.release-notes-1.0.0.0-rc1.md)
+- [Reporting Dashboards Plugin](https://github.com/opensearch-project/dashboards-reports/blob/main/release-notes/opensearch-dashboards-reports.release-notes-1.0.0.0-rc1.md)
+- [Trace Analytics Dashboards Plugin](https://github.com/opensearch-project/trace-analytics/blob/main/release-notes/opensearch-trace-analytics.release-notes-1.0.0.0-rc1.md)
+- [Gantt Charts Dashboards Visualization Plugin](https://github.com/opensearch-project/dashboards-visualizations/blob/main/release-notes/opensearch-dashboards-visualizations.release-notes-1.0.0.0-rc1.md)
+- [Async-Search](https://github.com/opensearch-project/asynchronous-search/blob/main/release-notes/opensearch-asynchronous-search.release-notes-1.0.0.0-rc1.md)
+- [Common-Utils](https://github.com/opensearch-project/common-utils/blob/main/release-notes/opensearch-common-utils.release-notes-1.0.0.0-rc1.md)
+
+**What does this Release Candidate not include?** 
+
+* *Minimum Artifacts* - Based on community feedback, we plan on releasing a standalone, minimal downloadable artifacts without the OpenSearch plugins (alerting, AD, security, etc…). We are expecting to deliver these artifacts by GA.
+* *Additional Artifacts* - In addition to the current artifacts (Linux tars and Docker images) we are hoping to deliver RPM (X64), Windows (X64), and DEB (X64) by GA and Tar (ARM64), DEB (ARM64), RPM (ARM64), MacOS (M1 ARM), and MacOS (X64) by OpenSearch 1.1. 
+* *Clients with License Checks* - We are aware of the issue with license checks which are causing incompatibilities (i.e. the recent release of Logstash & Beats 7.13) and are working on a solution for the community before GA.
+
+**How can you help?** 
+
+The best thing you can do is run the Release Candidate through its paces. Every bit of testing or feedback the community can provide helps ensure that the changes are doing what they should and we are building to the best end product. Couple specific places to start;
+
+- Install the Release Candidate and open an [issue](https://github.com/opensearch-project/OpenSearch/issues) to report any bugs you discover. 
+- Run rolling upgrades from existing version to OpenSearch RC1 in a sandbox cluster. 
+- Test clients! We want to know what is working (and not working) for you. 
+
```

Comment:
let's add one for use our docs and highlight any additional content you'd like to see or submit a PR and add any content you think would be valuable.

Reply:
Done!